database/migrations/2022_09_02_104041_create_posts_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained()->cascadeOnDelete();
```

Comment:
I think this means if the post is deleted, the user is also deleted? 😱

Reply:
actually no, this looks correct.

Reply:
yeah, the opposite. If the user is deleted, the post from the user are deleted too